src/bci_build/package/mariadb.py

```diff
@@ -129,8 +129,12 @@
 COPY healthcheck.sh /usr/local/bin/
 {DOCKERFILE_RUN} chmod 755 /usr/local/bin/healthcheck.sh
 
-COPY gosu /usr/local/bin/gosu
-{DOCKERFILE_RUN} chmod 755 /usr/local/bin/gosu
+COPY idexec /usr/local/bin/idexec
+{DOCKERFILE_RUN} chmod 755 /usr/local/bin/idexec
+
+# replace gosu calls with idexec
+{DOCKERFILE_RUN} sed -i 's/gosu/idexec/g' /usr/local/bin/{_ENTRYPOINT_FNAME}
```

Comment:
can we make this more robust by e.g. adding spaces or so around it? I'm nervous that it will replace some other occurence somewhere else and then we have a very difficult to trace bug

Reply:
e.g "s/ exec gosu / exec idexec /" 

would work for me.

Reply:
There are not many calls to gosu, nor anything with gosu in the middle of a word, but I get the point.

Reply:
> There are not many calls to gosu, nor anything with gosu in the middle of a word, but I get the point.

not today, but there might be some in the future